usaspending_api/awards/migrations/0058_reporting_matviews.py

```diff
@@ -0,0 +1,508 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.21 on 2019-08-09 15:43
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields
+import django.contrib.postgres.search
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('awards', '0057_auto_20190810_0246'),
```

Comment:
There's been a few new migrations in dev, so this file will have to be renamed to `0061_reporting_matviews.py` and change the dependency to `0060_transactionfpds_solicitation_date`

Reply:
Only if this PR is merged into dev. Current target is staging, which would require the migrations in dev to be altered